README.md

```diff
@@ -28,3 +32,21 @@ Welcome to the 4coder community repository.
 
 ## Mac (Untested)
 1. The steps should be the same as linux but replace the `*-linux.sh` with their `*-mac.sh` equivalents.
+
+## Build script prameter
+
+The build script accepts a parameter (mutually exclusive):
+- `/DDEV_BUILD` or `/DDEV_BUILD_X86` (default value) : build without optimizations.
+   Produces debug symbols.
+   Defines: `FRED_INTERNAL`, `FRED_SUPER` macros.
+- `/DOPT_BUILD` or `/DOPT_BUILD_X86` (similar to `build_optimized` script): build with optimizations.
+   Doesn't produce debug symbols.
+   Defines `FRED_SUPER` macro.
+- `/DPACKAGE_SUPER_X64` or `/DPACKAGE_SUPER_X86` (similar to `package` script): package 4coder for distribution.
+   Turns on optimizations.
```

Comment:
Oh the 4ed_build.cpp adds optimisation when you package with the package function, i never reaslised that. Here i was thinking I'd been using an unoptimsed core. God i hate this build system :D

Nice work documenting all this, superstar ⭐